src/tools/rbd/action/List.cc

```diff
@@ -241,8 +241,8 @@ int do_list(const std::string &pool_name, const std::string& namespace_name,
 	  }
 	  comp->name = i->name;
 	  comp->completion = new librbd::RBD::AioCompletion(nullptr, nullptr);
-	  r = rbd.aio_open_read_only(ioctx, comp->img, i->name.c_str(), nullptr,
-                                     comp->completion);
+	  r = rbd.aio_open_by_id_read_only(ioctx, comp->img, i->id.c_str(),
+                                           nullptr, comp->completion);
```

Comment:
@dillaman now `rbd ls -l` fails for v1 format images (catched by `cli_generic.sh:test_ls`).

Reply:
Ah ... thx